ons_alpha/jinja2/templates/base_page.html

```diff
@@ -36,7 +36,22 @@
 {#    {% primary_nav %}#}
 {#{% endblock header %}#}
 
-{% block preMain %}
-    {{ super() }}
-    {% include "templates/components/navigation/breadcrumbs.html" %}
+{% block pageContent %}
+    {# Adds a full-width section for the header area before the main page content #}
+    {# Changes the location of the <main> tag from the base template - ensures it surrounds the header area #}
+    <main id="main-content">
+        {# This will contain the breadcrumbs on individual templates #}
+        {% block header_area %}
+        {% endblock %}
```

Comment:
Shall we just have these on one line for tidiness? I don't see any reason to have a whitespace gap by default.

Reply:
Thanks @ababic - in a later PR I've added the disclaimer here so may as well leave it for now if that's ok.